pos_default_quantity/__openerp__.py

```diff
@@ -0,0 +1,25 @@
+# coding: utf-8
+# Copyright 2019 Coop IT Easy SCRLfs
+# 	    Robin Keunen <robin@coopiteasy.be>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Set Default Product Quantity in POS",
+    "version": "9.0.0.1.0",
+    "author": "Coop IT Easy SCRLfs, Odoo Community Association (OCA)",
+    "website": "www.coopiteasy.be",
+    "license": "AGPL-3",
+    "category": "Point of Sale",
+    "summary": """
```

Comment:
Use description key

Reply:
I changed it but the linting bot advises to remove description
```
pos_default_quantity/__openerp__.py:5: [C8103(manifest-deprecated-key), ] Deprecated key "description" in manifest file
```